release-notes/aws/v11.1.4.md

```diff
@@ -0,0 +1,31 @@
+# :zap: Giant Swarm Release v11.1.4 for AWS :zap:
+
+__Note:__ Upgrading to this release from any release prior v11.1.1 will cause a network downtime due to the network-related changes coming with the switch from Calico CNI to AWS CNI.
+
+## aws-operator [v8.3.0](https://github.com/giantswarm/aws-operator/releases/tag/v8.3.0)
+
+- Ensure that Pod subnet IP ranges don't overlap between tenant clusters, to enable VPC peering between tenant clusters in the same installation.
```

Comment:
We also added the VPC ID thingy in the CR status. Also not ensuring the CRD anymore happened here like in `cluster-operator`, which has a changelog for that in here.

Reply:
added